### PR TITLE
feat(menu-button): added custom aria-label selector for menu item button

### LINKED ISSUE #144: Menu-button: add custom aria-label selector.
Add an ability to set custom aria-label for menu buttons. E.g. Phone Input has menu items with both country and country code. In this case, we need the ability to exclude the country code as the country code is added on the textbox.

### PR DESCRIPTION
**Pull request by saiponnada**
Fixes #144 

Added `menuItemButtonLabelSelector` option to select the text for aria-label

### CODE REVIEW

File: packages/makeup-menu-button/dist/mjs/index.js [hunk 1]
```diff
@@ -112,8 +113,8 @@ function _onMenuItemSelect(e) {
   if (e.detail.el.getAttribute("role") !== "menuitemradio") {
     return;
   }
-  const icon = e.detail.el.querySelector(this._options.iconSelector).cloneNode(true);
-  const text = e.detail.el.innerText.trim();
+  const icon = e.detail.el.querySelector(this._options.menuItemIconSelector).cloneNode(true);
+  const text = this._options.menuItemButtonLabelSelector ? e.detail.el.querySelector(this._options.menuItemButtonLabelSelector)?.innerText.trim() : e.detail.el.innerText.trim();
   let content = this._buttonPrefix ? `${this._buttonPrefix} ${text}` : text;
   if (icon) {
     switch (this._options.buttonValueType) {
```
Comment (ianmcburnie) [inline packages/makeup-menu-button/dist/mjs/index.js:120]: This code might be easier to follow if we move the type "text" logic within the switch statement too...?

File: packages/makeup-menu-button/dist/mjs/index.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const defaultOptions = {
   buttonValueType: "text",
```
Comment (ianmcburnie) [inline packages/makeup-menu-button/dist/mjs/index.js:8]: I think one thing we might need to do is make it clearer that some of these options are only valid for the single select (i.e. role=menuitemradio) variant. Maybe we could move those into a sub mapping? Or any other ideas?

```
const defaultOptions = {
  customElementMode: false,
  expandedClass: "menu-button--expanded",
  menuSelector: ".menu-button__menu",
  buttonTextSelector: `.btn__text`,
  singleSelect: {
    buttonValueType: "text", // ["text", "icon", "both"],
    menuItemIconSelector: ".icon",
    menuItemButtonAriaLabelSelector: null,
  }
};
```

Comment (ianmcburnie) [reply packages/makeup-menu-button/dist/mjs/index.js:8]: Feel free to just create an issue for this and merge and we can come back to it later.

Comment (saiponnada) [reply packages/makeup-menu-button/dist/mjs/index.js:8]: Created a new issue - https://github.com/makeup/makeup-js/issues/146

**Pull request merged**